tests/mir-opt/const_prop/return_place.add.ConstProp.diff

```diff
@@ -6,9 +6,8 @@
       let mut _1: (u32, bool);             // in scope 0 at $DIR/return_place.rs:+1:5: +1:10
   
       bb0: {
--         _1 = CheckedAdd(const 2_u32, const 2_u32); // scope 0 at $DIR/return_place.rs:+1:5: +1:10
+          _1 = CheckedAdd(const 2_u32, const 2_u32); // scope 0 at $DIR/return_place.rs:+1:5: +1:10
 -         assert(!move (_1.1: bool), "attempt to compute `{} + {}`, which would overflow", const 2_u32, const 2_u32) -> bb1; // scope 0 at $DIR/return_place.rs:+1:5: +1:10
-+         _1 = const (4_u32, false);       // scope 0 at $DIR/return_place.rs:+1:5: +1:10
 +         assert(!const false, "attempt to compute `{} + {}`, which would overflow", const 2_u32, const 2_u32) -> bb1; // scope 0 at $DIR/return_place.rs:+1:5: +1:10
```

Comment:
I'm guessing checked math doesn't work yet?

Reply:
It does work. All uses of `_1.0` and `_1.1` are propagated. However, we don't replace the `CheckedAdd` rvalue with the evaluated tuple.